index.html

```diff
@@ -155,14 +161,14 @@ <h2 class="contact-me-title">I'm always interested in hearing about new projects
     </li>
 </ul>
 </form>
-<ul class="mediaimgs">
-    <li><i><img src="./Images/twitter.png" alt="twitter" height="20" width="22"/></i></li>
-    <li><i><img src="./Images/linkedin.png" alt="linkedin" height="20" width="22"/></i></li>
-    <li><i><img src="./Images/medium.png" alt="medium" height="20" width="22"/></i></li>
-    <li><i><img src="./Images/github.png" alt="github" height="20" width="22"/></i></li>
-    <li><i><img src="./Images/angellist.png" alt="angellist" height="20" width="22"/></i></li>
+<ul class="mediaimgs1">
+    <li class="mediaimg1"><i><img src="./Images/twitter.png" alt="twitter" height="20" width="22"/></i></li>
+    <li class="mediaimg2"><i><img src="./Images/linkedin.png" alt="linkedin" height="20" width="22"/></i></li>
+    <li class="mediaimg3"><i><img src="./Images/medium.png" alt="medium" height="20" width="22"/></i></li>
+    <li class="mediaimg4"><i><img src="./Images/github.png" alt="github" height="20" width="22"/></i></li>
+    <li class="mediaimg5"><i><img src="./Images/angellist.png" alt="angellist" height="20" width="22"/></i></li>
```

Comment:
-Here is an opportunity to use ```<footer>``` tags that could be divided into different section

Reply:
Perfect it will improve my code, thank you!